src/Mgate/SuiviBundle/Controller/EtudeController.php

```diff
@@ -232,12 +229,22 @@ public function modifierAction(Request $request, Etude $etude)
                 $em->persist($etude);
                 $em->flush();
 
-                return $this->redirect($this->generateUrl('MgateSuivi_etude_voir', array('nom' => $etude->getNom())));
+                if ($request->get('ap')) {
+                    return $this->redirect($this->generateUrl('MgateSuivi_ap_rediger', array('id' => $etude->getId())));
+                } else {
+                    return $this->redirect($this->generateUrl('MgateSuivi_etude_voir', array('nom' => $etude->getNom())));
+                }
+            } else {
```

Comment:
Unnecessary else (the condition above always return, so no need to add an else block around the following code)

Should be the same in `modifierAction`